billing/billing.proto

```diff
@@ -548,7 +558,7 @@ service CurrencyService {
   rpc GetExchangeRate (GetExchangeRateRequest)
       returns (GetExchangeRateResponse) {
     option (google.api.http) = {
-      get: "/billing/currencies/rates/{from}/{to}"
+      get: "/billing/currencies/rates/{from.name}/{to.name}"
```

Comment:
Why did you rename the variable?

Reply:
Since I changed "Currency" type from primitive to Message I had to change URL mappings. But yeah, my example will not work, so I either move those mappings from URL Path to URL queries or to Body, which one should I choose?